lib/puppet/pops/parser/eparser.rb

```diff
@@ -1,6 +1,6 @@
 #
 # DO NOT MODIFY!!!!
-# This file is automatically generated by Racc 1.5.2
+# This file is automatically generated by Racc 1.4.16
```

Comment:
Could you try updating racc and generating the eparser.rb? IIRC newer racc versions fixed a bug so the parser output is stable.

Reply:
Updated! I didn't realize the machinations necessary to actually install `racc` as part of the bundle.